v4/source/cloud.yaml

```diff
@@ -319,3 +319,26 @@
           $ref: "#/components/responses/Forbidden"
         "501":
           $ref: "#/components/responses/NotImplemented"
+  /cloud/subscription/stats:
+    get:
+      tags:
+        - cloud
+      summary: GET endpoint for cloud subscription stats
+      description: >
+        An endpoint for that returns stats about a users subscription. For example remaining seats on a free tier
+
+        ##### Permissions
+
+        This endpoint should only be accessed in a Mattermost Cloud instance
+
+        __Minimum server version__: 5.30
+        __Note:__ This is intended for internal use and is subject to change.
+      responses:
```

Comment:
What about 401/403 responses if the permissions are not right?

Reply:
Hello @ethervoid , thanks a lot for the review. This endpoint doesn't require any permissions.